### PR TITLE
[Master] Remove AuthnDataPublisherProxy event listener config as it is available by default

### PR DESCRIPTION
**Pull request by ashendes**

### CODE REVIEW

File: en/docs/learn/configuring-identity-analytics.md [hunk 1]
```diff
@@ -17,14 +17,6 @@ To do this, the developers need to view authentication statistics about the logi
 ## Enable analytics
 
 Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and add the following event listeners to enable analytics in WSO2 Identity Server.
```
Comment (mefarazath) [inline en/docs/learn/configuring-identity-analytics.md:19]: Shouldn't this now say 'enable the following event publishers to enable analytics'

Comment (ashendes) [reply en/docs/learn/configuring-identity-analytics.md:19]: Isn't this this a bit too long for a heading? 
@gomathyk thoughts?

Comment (mefarazath) [reply en/docs/learn/configuring-identity-analytics.md:19]: Oops. I didn't mean the heading.

I actually suggested a change in the text in the first sentence after the heading

Comment (ashendes) [reply en/docs/learn/configuring-identity-analytics.md:19]: Ah yes. I see what you mean

File: en/docs/learn/configuring-identity-analytics.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ To do this, the developers need to view authentication statistics about the logi
 
 ## Enable analytics
 
-Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and add the following event listeners to enable analytics in WSO2 Identity Server. 
+Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event listeners to enable analytics in WSO2 Identity Server.
```
Comment (mefarazath) [inline en/docs/learn/configuring-identity-analytics.md:19]:
```suggestion
Open the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the following event publishers to enable analytics in WSO2 Identity Server. 
```

**Review by mefarazath — APPROVED** [review]

**Pull request merged**